kernel/selinux/rules.c

```diff
@@ -22,7 +22,7 @@ static struct policydb *get_policydb(void)
 {
 	struct policydb *db;
 // selinux_state does not exists before 4.19
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 9, 337)
+#ifdef USE_SELINUX_STATE
```

Comment:
And also, don't rely on this MARCRO on GKI kernels (version bigger than 5.10)

Reply:
https://github.com/tiann/KernelSU/pull/402/commits/e4f43d1b605a1717f3993723497b81af34963508 ignored on 5.10+.
Maybe we can further optimize out the grep check in Makefile for 5.10+ by checking the `VERSION`/`PATCHLEVEL` which are set by the kernel root Makefile, but I am not quite familiar with Makefile syntax.

Reply:
https://github.com/tiann/KernelSU/pull/402/commits/36909422654b36db74e0574e3ead907697098cc8 fixed.